eventuals/errors.h

```diff
@@ -12,11 +12,13 @@ namespace eventuals {
 // All errors should derive from 'Error' and be copyable
 // since 'std::make_exception_ptr' works with a copy argument
 // and there is no way to move error in it.
-class Error {
+class Error : public std::exception {
  public:
   virtual ~Error() = default;
 
-  virtual std::string what() const noexcept = 0;
+  const char* what() const noexcept override {
+    return "Eventuals inner 'Error'";
```

Comment:
Perhaps you mean `internal` instead of `inner`?
Internal = it happened inside Eventuals
Inner = it is wrapped inside some outer thing (and I don't think there's an outer 'Error'?)